### PR TITLE
#584 - PHP deprecated warning

### PR DESCRIPTION
**Pull request by Xavjer**

### CODE REVIEW

File: lib/functions/filedir/function.makeCorrectDir.php [hunk 1]
```diff
@@ -27,9 +27,9 @@
 function makeCorrectDir($dir) {
 
 	if (version_compare("5.4.6", PHP_VERSION, ">")) {
-		assert('is_string($dir) && strlen($dir) > 0 /* $dir does not look like an actual folder name */');
```
Comment (d00p) [inline lib/functions/filedir/function.makeCorrectDir.php]: the comment was part of the assertion on purpose as there was no description parameter until 5.4.8 and so the comment would be output as well

Comment (Xavjer) [reply lib/functions/filedir/function.makeCorrectDir.php]: > the comment was part of the assertion on purpose as there was no description parameter until 5.4.8 and so the comment would be output as well

Ah, that explains why this was in ' all the time. This should therefore be fine now. I tested the else clause on my server and, at least for 7.2 it is now fine. Did not find any errors after that.

Comment (d00p) [reply lib/functions/filedir/function.makeCorrectDir.php]: So, if you could change that back for <=5.4.6 I'd gladly merge

**Comment by d00p** [comment]
Argh....next time, please only against MASTER branch...not 0.9.x

**Comment by Xavjer** [comment]
That's why I made the change in 0.9.x ':)
I was unsure if the 0.9.x will be used in the future and therefore wanted to evade that this happens again ;)

**Comment by d00p** [comment]
it will only be used for critical security bugfix releases in the 0.9 series....but we want to focus on the api based 0.10.0 now

**Pull request merged**